packages/http/package.json

```
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@stoplight/json": "^3.13.7",
+    "@stoplight/json-schema-merge-allof": "^0.7.8",
```

```suggestion
    "@stoplight/json-schema-merge-allof": "0.7.8",
```
perhaps let's lock the version, as according to semantic versioning, 0.x.x is considered as "under active development" and the public API may change anytime. I'm not saying it'll change, but at the same time I'd not be particularly surprised if we introduced some subtle breaking changes that could affect the work you did here.